docs/source/tutorials/sklearn_svc.ipynb

```diff
@@ -0,0 +1,303 @@
+{
```

Comment:
<p>better visualize the circuit using c.draw()</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tencent-quantum-lab/tensorcircuit/pull/164/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
Due to unknown error, c.draw() cannot be added under the function where the circuit is created.

Reply:
it is ok now since the circuit structure is more clearly organized